lightpath/tests/conftest.py

```diff
@@ -1,16 +1,17 @@
-import os.path
 import logging
+import os.path
 from types import SimpleNamespace
 
-import happi
 import pytest
-from ophyd import Device, Kind, Component as Cpt
-from ophyd.signal import AttributeSignal
-from ophyd.status import DeviceStatus
-from ophyd.utils import DisconnectedError
 
+import happi
 import lightpath
 from lightpath import BeamPath
+from ophyd import Component as Cpt
+from ophyd import Device, Kind
+from ophyd.signal import AttributeSignal
+from ophyd.status import DeviceStatus
+from ophyd.utils import DisconnectedError
```

Comment:
This is again the wrong sort order, unless I misunderstand the rules? I'm going to stop and look them up again...

Reply:
Yes, this is definitely wrong. Stuff like `ophyd` are third-party imports, not local imports. https://www.python.org/dev/peps/pep-0008/#imports